guardian/shortcuts.py

```diff
@@ -159,7 +170,7 @@ def get_perms_for_model(cls):
     return Permission.objects.filter(content_type=ctype)
 
 def get_users_with_perms(obj, attach_perms=False, with_superusers=False,
-        with_group_users=True):
+        with_group_users=True, direct_perms_only=False):
```

Comment:
Isn't `direct_perms_only` redundant? You could simply set `with_superusers` and `with_group_users` to `False` and you would get the same result as setting `direct_perms_only` to `True`? No?